test/spec/index.spec.js

```diff
@@ -14,33 +16,147 @@ describe('index.js', function() {
       var parsedOptions = krabby._getOptions(options);
       assert(_.isEqual(options, parsedOptions));
     });
+    it('finds a krabby config when found inside of another object', function() {
```

Comment:
Yea, that's hard to state... What about `it uses the config.krabby property if available`?